lib/Elevate/SystemctlService.pm

```diff
@@ -86,13 +86,17 @@ sub remove ($self) {
 
 sub stop ($self) {
 
+    return unless $self->is_active;
```

Comment:
these things should not be altered. There is no need with the touch file approach.

Reply:
I m fixing an existing bug, glad to submit it as its own

Reply:
submitted via https://github.com/cpanel/elevate/pull/309